lib/cmd/import.js

```diff
@@ -203,6 +203,23 @@ function parseDispatch(str, url) {
     .flatMap((dispatch) => prefixes.add(dispatch, prefix));
 }
 
+/**
+ * parse a json object that is a direct translation
+ * of a bootstrap yaml file returning a string
+ * of prefixed dispatches
+ * @param  {object} obj
+ * @param  {string} url
+ * @return {Stream}
+ */
+function parseBootstrapJson(obj, url) {
+  const prefix = config.get('url', url);
+
+  return h.of(obj)
+    .flatMap(obj => formatting.toDispatch(h.of(obj)))
+    .flatMap(dispatch => prefixes.add(dispatch, prefix));
+}
```

Comment:
this function doesn't do what its name implies. It looks like it converts an object literal (not JSON) and converts it to an array of dispatches